pueue_lib/src/process_helper/mod.rs

```diff
@@ -83,16 +84,35 @@ pub fn compile_shell_command(settings: &Settings, command: &str) -> Command {
     }
 
     let executable = compiled_command.remove(0);
+    let metadata = std::fs::metadata(&executable);
+    if !std::path::PathBuf::from(&executable).is_file() {
+        warn!(
+            message = "Couldn't sanity check that the executale is a real file",
+            ?executable,
+            ?metadata
+        );
+    } else {
+        trace!(
+            message = "Command sanity check has a file-like executable",
+            ?executable,
+            ?metadata
+        );
+    }
 
     // Chain two `powershell` commands, one that sets the output encoding to utf8 and then the user provided one.
+    debug!(
+        message = "Building command in preperation to spawn",
```

Comment:
It's already prepared at this point in time. Maybe something like this:

```suggestion
        message = "Prepared command before spawn",
```

Reply:
I just moved this below to use `std::os::Command`'s impl of Debug which was actually more helpful to me (it includes cool things like `ENV_VARS=value "executable escaped" "with args"`